src/Query/Mysql/RegexpReplace.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace DoctrineExtensions\Query\Mysql;
+
+use Doctrine\ORM\Query\AST\Functions\FunctionNode;
+use Doctrine\ORM\Query\Lexer;
+
+/**
+ * @example SELECT REGEXP_REPLACE(string, search, replace)
+ * @link https://www.postgresql.org/docs/current/functions-matching.html#FUNCTIONS-POSIX-TABLE
```

Comment:
:thinking:

Reply:
You are right, I should have looked better when copying the test from postgresql